webknossos/pyproject.toml

```diff
@@ -75,16 +75,13 @@ all = ["pims","tifffile","imagecodecs","JPype1","pylibCZIrw"]
 
 [tool.poetry.dev-dependencies]
 # autoflake
-black = "^23.1.0"
 dask = { version = "^2023.9.1", python = ">=3.9", extras = ["distributed"] }
 hypothesis = "^6.35.0"
 icecream = "^2.1.1"
 inducoapi = "^2.0.2"
-isort = "^5.9.3"
 jsonschema = "^4.6.0"
-mypy = "^1.0.0"
```

Comment:
We were running `mypy 1.6.x` anyway, so I fixed that as a minimum requirement. The current version is 1.9.x.